chart/skywalking/templates/oap-clusterrolebinding.yaml

```diff
@@ -31,4 +31,9 @@ subjects:
 - kind: ServiceAccount
   name: {{ template "skywalking.serviceAccountName.oap" . }}
   namespace: {{ .Release.Namespace }}
+{{- if .Values.satellite.enabled }}
```

Comment:
Why does the satellite need the cluster role? Which resources it has to get?

Reply:
It needs to get `pod` resources because the satellite needs to get all `oap` pod grpc ports to forward content.

Reply:
From the oap and satellite's manifests, they would be in the same namespace, which means a role with getting/listing pods is enough to satellite instead of a cluster role.

Reply:
Sure, should I create a new role to do this? I see we already have the role so reuse them.

Reply:
sure. I feel free to add a new one